core/store/src/metrics.rs

```diff
@@ -236,6 +237,91 @@ pub static COLD_COPY_DURATION: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static GET_STATE_PART_WITH_FS_ELAPSED: Lazy<Histogram> = Lazy::new(|| {
```

Comment:
Two comments:
* does histogram allow to track the **sum** of all captured times? I think it is more useful for identifying bottlenecks.
* we need a label `shard_id` for all these metrics

Reply:
Yes, histograms export:
* Count per bucket
* Sum
* Count

Added `shard_id` to all metrics.